### DISCUSSION TITLE
How do I customize the default Identity UI?

### DISCUSSION THREAD

**Discussion by maxi512** [jasontaylordev/CleanArchitecture#1249] [General]
How do I customize the default Identity UI?

**Comment by huenchunb** [comment]
I have the same question

**Comment by jasontaylordev** [comment]
First, you need to scaffold ASP.NET Core Identity, and then you can customise it. Learn more here; https://learn.microsoft.com/en-us/aspnet/core/security/authentication/scaffold-identity?view=aspnetcore-8.0&tabs=visual-studio.
